### PR TITLE
Add Voice Activity Detection scripts, tests, tutorials

### PR DESCRIPTION
**Pull request by fayejf**
Doc for VAD will be on very soon.
Current stage:  Improving voice activity detection for streaming inference (VAD_v2)

### CODE REVIEW

File: examples/asr/configs/quartznet_vad_3x1.yaml [hunk 1]
```diff
@@ -0,0 +1,123 @@
+model: "QuartzNet"
+sample_rate: &sample_rate 16000
+dropout: &drop 0.0
+repeat:  &rep  1
+augment: true
+lr_schedule: "PolynomialHoldDecayAnnealing"
+lr_warmup_proportion: 0.1
+kernel_size_factor: &kfactor 1.0
+
+AudioToSpeechLabelDataLayer:
```
Comment (tkornuta-nvidia) [inline examples/asr/configs/quartznet_vad_3x1.yaml:10]: import_from_config -> new configuration format...

https://github.com/NVIDIA/NeMo/blob/5d1527aa940591878acb9745a66d87ddb50a76c9/examples/asr/configs/jasper_an4.yaml#L10

File: examples/asr/notebooks/3_Speech_Commands_using_NeMo.ipynb [hunk 2]
```diff
@@ -2,9 +2,96 @@
  "cells": [
```
Comment (blisc) [inline examples/asr/notebooks/3_Speech_Commands_using_NeMo.ipynb:2]: Why does this file have a bunch of error messages now?

File: examples/asr/quartznet_vad.py [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+import nemo.collections.asr as nemo_asr
+import nemo.utils.argparse as nm_argparse
+from nemo.collections.asr.helpers import (
+    monitor_classification_training_progress,
+    process_classification_evaluation_batch,
+    process_classification_evaluation_epoch,
+)
+from nemo.utils.lr_policies import CosineAnnealing, PolynomialDecayAnnealing, PolynomialHoldDecayAnnealing
+
+logging = nemo.logging
```
Comment (blisc) [inline examples/asr/quartznet_vad.py:35]: please use the new `from nemo.utils import logging`

File: examples/asr/quartznet_vad.py [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+    crop_pad_augmentation = nemo_asr.CropOrPadSpectrogramAugmentation(audio_length=128)
+
+    N = len(data_layer)
+    steps_per_epoch = math.ceil(N / (args.batch_size * args.iter_per_step * args.num_gpus))
+    logging.info('Steps per epoch : {0}'.format(steps_per_epoch))
+    logging.info('Have {0} examples to train on.'.format(N))
+
+    #     data_preprocessor = nemo_asr.AudioToMelSpectrogramPreprocessor(
+    #         sample_rate=sample_rate, **jasper_params["AudioToMelSpectrogramPreprocessor"],
+    #     )
```
Comment (blisc) [inline examples/asr/quartznet_vad.py:134]: Please delete commented out code

**Comment by fayejf** [comment]
Close this PR to remove unexpected git commit history and open a new one.

**Pull request closed without merging**